styles.css

```diff
@@ -0,0 +1,116 @@
+* {
+  box-sizing: border-box;
+}
+.sr-only{
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+}
+.grid-container {
+  display: grid;
+  grid-gap: 2px;
+  grid-template-columns: auto auto auto auto;
+  grid-template-rows: auto auto auto auto;
+  justify-content: space-evenly;
+}
+a:link, a:visited {
+  color: white;
+}
+a:link {
+  text-decoration: none;
+}
+.grid-container {
+  display: grid;
+  grid-gap: 2px;
+  grid-template-columns: auto auto auto auto;
+  grid-template-rows: auto auto auto auto;
+  justify-content: space-evenly;
+}
+.grid-item-header{
+  grid-column: 1 / 3;
+  grid-row: 1/4;
+  background: #03a3df;
+  padding: 10px;
+  background: #000000 url(images/header-image.jpg);
+}
+.header{
```

Comment:
See if you can use this without float. We want to minimize floats as we have better tools. Try flexbox, grid or position.